src/main/java/com/twilio/rest/fax/v1/FaxReader.java

```diff
@@ -208,15 +210,29 @@ private void addQueryParams(final Request request) {
         }
 
         if (dateCreatedOnOrBefore != null) {
-            request.addQueryParam("DateCreatedOnOrBefore", dateCreatedOnOrBefore.toString());
+            request.addQueryParam("DateCreatedOnOrBefore", toZulu(dateCreatedOnOrBefore));
         }
 
         if (dateCreatedAfter != null) {
-            request.addQueryParam("DateCreatedAfter", dateCreatedAfter.toString());
```

Comment:
Looks like using `toOffsetDateTime()` first is simpler: https://stackoverflow.com/a/43635374/1660457